From 6e5c2f158826efeafe48803244b98c6fa7d35d52 Mon Sep 17 00:00:00 2001
From: BambooIV <ningdisheng@gmail.com>
Date: Tue, 15 Mar 2016 12:22:33 +0800
Subject: [PATCH 14/36] nx503a: ARM: dts: msm: set overlap fudge factor to 1.75
 for msm8974

cherry-pick from 19ac5873521aa17a59646c677e86665509c2d722

Set the fudge factor for the overlap bandwidth to 1.75.
This fixes underruns observed in some use cases with 8974 targets.
---
 arch/arm/boot/dts/NX503A/msm8974-mdss.dtsi | 1 +
 1 file changed, 1 insertion(+)

diff --git a/arch/arm/boot/dts/NX503A/msm8974-mdss.dtsi b/arch/arm/boot/dts/NX503A/msm8974-mdss.dtsi
index 1615de6..fe40056 100644
--- a/arch/arm/boot/dts/NX503A/msm8974-mdss.dtsi
+++ b/arch/arm/boot/dts/NX503A/msm8974-mdss.dtsi
@@ -35,6 +35,7 @@
 		qcom,mdss-ab-factor = <5 2>;		/* 2 times    */
 		qcom,mdss-ib-factor = <3 2>;		/* 1.2 times  */
 		qcom,mdss-clk-factor = <5 3>;		/* 1.25 times */
+		qcom,mdss-ib-factor-overlap = <7 4>;	/* 1.75 times  */
 
 		qcom,max-clk-rate = <320000000>;
 		qcom,mdss-pipe-vig-off = <0x00001200 0x00001600
-- 
1.9.1

